hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -854,6 +857,28 @@ ContainerCommandResponseProto handleWriteChunk(
       chunkManager
           .writeChunk(kvContainer, blockID, chunkInfo, data, dispatcherContext);
 
+      final boolean isCommit = dispatcherContext.getStage().isCommit();
+      if (isCommit && writeChunk.hasBlock()) {
```

Comment:
Shall we check COMBINED_PUTBLOCK_WRITECHUNK_RPC is finalized here?

Reply:
COMBINED_PUTBLOCK_WRITECHUNK_RPC doesn't have the concept of "finalize". Once the runtime is updated, the datanode version becomes COMBINED_PUTBLOCK_WRITECHUNK_RPC.

Reply:
which should make sense because this PR does not change on-disk format.